File: Tests/SkywalkerOCTests/UIKitDevelop/UIImage/UIImageTests.m
//
//  UIImageTests.m
//  
//
//  Created by 杨俊艺 on 2022/2/16.
//

#import <XCTest/XCTest.h>
@import SkywalkerOC;

@interface UIImageTests : XCTestCase

@end

@implementation UIImageTests

- (void)setUp {
    
}

- (void)tearDown {
    
}

- (void)testExample {
    UIImage *i = [UIImage systemImageNamed:@"cc"];
    [i convertToGrayImage];
}

- (void)testPerformanceExample {
    [self measureBlock:^{
        
    }];
}

@end

File: Tests/SkywalkerTests/UIKitDevelop/UIImage/UIImageTests.swift
//
//  UIImageTests.swift
//
//
//  Created by 杨俊艺 on 2022/2/16.
//

@testable import Skywalker
@testable import SkywalkerOC
import XCTest

class UIImageTests: XCTestCase {
    func testExample() throws {
        let ii = UIImage()
        ii.convertToGray()
        ii.color(atPixel: CGPoint.zero)
    }
}
